### PR TITLE
Removed old example app, created new example app project with resused…

### PR DESCRIPTION
**Pull request by jerzyKleszcz**
… identifiers and certificates from old example, configured initial dependecies.

#190

### CODE REVIEW

File: examples/Sniffator/android/app/build.gradle [hunk 1]
```diff
@@ -88,7 +99,7 @@ android {
 
     defaultConfig {
         applicationId "com.polidea.sniffator"
-        minSdkVersion 18
+        minSdkVersion 16
```
Comment (Cierpliwy) [inline examples/Sniffator/android/app/build.gradle]: Go back to 18.

File: examples/Sniffator/android/app/build.gradle [hunk 2]
```diff
@@ -136,13 +147,11 @@ dependencies {
     compile fileTree(dir: "libs", include: ["*.jar"])
     compile "com.android.support:appcompat-v7:23.0.1"
     compile "com.facebook.react:react-native:+"  // From node_modules
-    compile project(':react-native-ble-plx')
```
Comment (Cierpliwy) [inline examples/Sniffator/android/app/build.gradle]: I think that we need to configure `react-native-ble-plx` once again. It is up to you if you would like to do it in separate commit.

File: examples/Sniffator/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -5,12 +5,10 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
 
     <uses-sdk
-        android:minSdkVersion="18"
-        android:targetSdkVersion="23" />
+        android:minSdkVersion="16"
```
Comment (Cierpliwy) [inline examples/Sniffator/android/app/src/main/AndroidManifest.xml]: 18

File: examples/Sniffator/android/app/build.gradle (hunk 2 above)
Comment (jerzyKleszcz) [reply examples/Sniffator/android/app/build.gradle]: Done, thanks for pointing that!
